src/Infrastructure/Aggregate/PreconditionsTree/ActiveDirIsReady.php

```diff
@@ -13,10 +13,7 @@ public function __construct(
         ActiveDirExistsInterface $activeDirExists,
         ActiveDirIsWritableInterface $activeDirIsWritable,
     ) {
-        /** @var array<\PhpTuf\ComposerStager\Domain\Service\Precondition\PreconditionInterface> $children */
-        $children = func_get_args();
-
-        parent::__construct(...$children);
+        parent::__construct($activeDirExists, $activeDirIsWritable);
```

Comment:
I'll explain my rationale for this in the main conversation so we can discuss it in one place.